Utility.cpp

```diff
@@ -31,6 +32,19 @@ std::vector<std::unique_ptr<Item>> makeDefensiveItems(int num)
     return items;
 }
 
+std::vector<std::unique_ptr<Item>> makeAttackItems(int num)
+{
+    std::vector<std::unique_ptr<Item>> items;
+    
+    while( num-- >= 0 )
+    {
+        items.push_back( std::unique_ptr<AttackItem>(new AttackItem()) );
+    }
+    
+    std::cout << "made " << items.size() << " attack items" << std::endl;
+    return items;
+}
+
```

Comment:
This function is not needed. 
Dragon Slayers get ONE attack item, not a vector of them
- [ ] revise